### PR TITLE
Fix post header text overlap

### LINKED ISSUE #701: Users with long names overlap location text
env: [https://staging.fightpandemics.work/feed](https://staging.fightpandemics.work/feed)
This issue is only seen on mobile.

Users with short names, there is no wrapping issue:
IMG_4903

Users with long names, the name wraps and overlaps the location text:
IMG_4904

**Expected:** Move location text lower if name text wraps

### PR DESCRIPTION
**Pull request by jyclam**
### What does this pull request aim to achieve? 💯
Fixes post author name overlapping with location

## Before
Screen Shot 2020-07-16 at 2 36 34 AM

Simulated Pixel2XL
Screen Shot 2020-07-16 at 2 36 20 AM

## After
Screen Shot 2020-07-16 at 2 34 47 AM

Simulated Pixel2XL
Screen Shot 2020-07-16 at 2 35 23 AM

Simulated iPhone 5/SE
Screen Shot 2020-07-16 at 2 44 14 AM

_Please be concise and link any related issue(s):_
closes #701

### CODE REVIEW

File: client/src/components/Feed/Post.js [hunk 1]
```diff
@@ -339,21 +339,27 @@ const Post = ({
 
   const renderHeader = (
     <Card.Header
-      title={post?.author?.name}
+      title={
+        <div className="title-Wrapper">
```
Comment (idjevm) [inline client/src/components/Feed/Post.js:343]: need to update to Kebab case here on line 343 too!

Comment (idjevm) [inline client/src/components/Feed/Post.js:343]: need to update to Kebab case here on line 343 too!

Comment (jyclam) [reply client/src/components/Feed/Post.js]: Fixed!

**Comment by jyclam** [comment]
Hmm @rabbi08 I'm not actually able to reproduce it from the dev tool emulator.  414 x 896 is the viewport size of iPhone 11 according to [this site](https://docs.adobe.com/content/help/en/target/using/experiences/vec/mobile-viewports.html. ) but it looks fine in the Chrome emulator.

Screen Shot 2020-07-21 at 10 29 35 AM

Nonetheless, I've made a change that should fixed it and also refactored to handle long city names. Can you re-test on your phone and see if it is fixed?

**Comment by rabbi08** [comment]
> Hmm @rabbi08 I'm not actually able to reproduce it from the dev tool emulator. 414 x 896 is the viewport size of iPhone 11 according to [this site](https://docs.adobe.com/content/help/en/target/using/experiences/vec/mobile-viewports.html.) but it looks fine in the Chrome emulator.
> 
> Screen Shot 2020-07-21 at 10 29 35 AM
> 
> Nonetheless, I've made a change that should fixed it and also refactored to handle long city names. Can you re-test on your phone and see if it is fixed?

@jyclam 
I have retested the issue, iPhone 11 Pro Max, the issue still exist. P.S. the screenshot. Sorry for miscommunication on the device I mention before, its iPhone 11 Pro max not iPhone 11.

image

**Comment by jyclam** [comment]
Okay, thanks @rabbi08 I will investigate further.

**Comment by rabbi08** [comment]
> Okay, thanks @rabbi08 I will investigate further.

@jyclam 
Just thought to share, that while testing #1246, I have observed that this issue does NOT exist.

**Comment by jyclam** [comment]
thanks @rabbi08, I think the original issue only presented if the author name is long or if the device viewport is narrow (or a combination of the two) like here: https://featurehorizontal-rule-between-posts.fightpandemics.xyz/organisation/5f0c94cbee215d0011c9a134

**Comment by galeza** [comment]
Tested on xiaomi redmi note 8:
Screenshot_2020-07-23-12-47-11-344_com android chrome
Screenshot_2020-07-23-12-46-19-320_com android chrome

**Pull request merged**